library/ssl_tls13_client.c

```diff
@@ -1440,11 +1440,124 @@ static int ssl_tls13_read_server_hello( mbedtls_ssl_context *ssl )
 }
 
 /*
- * Handler for MBEDTLS_SSL_ENCRYPTED_EXTENSIONS
+ *
+ * EncryptedExtensions message
```

Comment:
```suggestion
 * Handler for MBEDTLS_SSL_ENCRYPTED_EXTENSIONS
 *
 * EncryptedExtensions message
```

Reply:
Please keep `Handler for MBEDTLS_SSL_ENCRYPTED_EXTENSIONS` , that is to avoid commit conflict .

If preceding PR was changed , here won't be conflict with the line

Reply:
If all depended PRs were merged , it can be remove

Reply:
If I leave this kind of handlers there, it will report error when compiling, cause we just define but will not use it.

Reply:
Rebase to latest dummy handler PR, and rename the xxx_read_xxx() to xxx_process_xxx().